### PR TITLE
Enable sponsor button, improve issue templates

### PR DESCRIPTION
**Pull request by TheRealPear**
I wanted to give back to ZipCaptions in ways that I could, thank you for everything you've done. Here's a summary of the changes I made:
* Switch to the newer & better [issue form templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) that has a strict layout compared to the old ones.
* Enabled the [sponsor button](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository) and linked ZipCaptions' Patreon and BuyMeCoffee, this should increase exposure since I didn't know about them when I first started using this app.
* For some reason, GitHub didn't detect the repo having a code of conduct file in the [community standards](https://github.com/jptrsn/zip-captions/community) page. My guess is that there's a whitespace issue with the Markdown file.
* Cleaned up the readme file a little bit and corrected some grammatical issues

## Summary by CodeRabbit

- **New Features**
	- Added funding options through Patreon and Buy Me a Coffee.
	- Introduced new templates for bug reports and feature requests.
	- Added a link to the ZipCaptions Discord server for community support.
	- Implemented a Contributor Covenant Code of Conduct to ensure a respectful community environment.

- **Bug Fixes**
	- Removed outdated bug report and feature request templates.

- **Documentation**
	- Updated the README.md for clarity and consistency.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      label: Device model
+      description: 'e.g.: iPhone 6'
+  - type: input
+    id: device-os-input
+    attributes:
+      label: Operating system & version
+      description: >-
+        Desktop e.g.: Windows, macOS, Linux (with distro name), ChromeOS. Mobile
+        e.g.: iOS/iPadOS, Android
+    validations:
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:66]: Line breaks are not supported in the description field and including the version number is already implied.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+  - type: dropdown
+    id: device-type-dropdown
+    attributes:
+      label: Which device did the bug occur on?
+      options:
+        - Desktop/Laptop
+        - Smartphone/Tablet
+        - Other or N/A
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:51]: Hilarious, but very unlikely. `Other or N/A` field already exist for this.

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ We currently support multiple languages, by navigating to the `Settings` page.
 Contributions are welcome! Anything from bug reports to feature enhancements to pull requests are welcomed and encouraged. If you have questions about using the application, we recommend visiting the [Discord Server](https://discord.gg/Swe2JeHnPc) and asking your questions there.
 
 ## Languages
-ZipCaptions supports multiple languages, to the best of our ability. If you notice a spelling or grammatical issue, feel free to adjust the text and submit a pull request (guidelines below). For additional language support, you are welcome to create a new translation file,
+ZipCaptions supports multiple languages to the best of our ability. If you notice a spelling or grammatical issue, feel free to adjust the text and submit a pull request (guidelines below). For additional language support, you are welcome to create a new translation file,
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply README.md]: This has been fixed.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply .github/ISSUE_TEMPLATE/config.yml:1]: This is pretty common, there's always a situation where either template might not work the best for reporting something.

File: README.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 # ZipCaptions
+**Free Speech-To-Text In Your Browser**
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply README.md:2]: This undo my change and adds an unnecessary entry to the readme's table of content.

**Review by jptrsn — APPROVED** [review]
Thanks for making these changes! I appreciate it a lot.

**Comment by TheRealPear** [comment]
No problem. I forgot to mention that you have to enable the sponsor button in repo setting, this pull request just configures it so GitHub knows what to show.

**Pull request merged**